masq_lib/src/messages.rs

```diff
@@ -843,6 +845,12 @@ pub struct UiWalletAddressesResponse {
 }
 conversation_message!(UiWalletAddressesResponse, "walletAddresses");
 
+#[derive(Message, Clone, PartialEq, Eq)]
```

Comment:
Look at the other message in this file and notice that you don't find any Actor messages here. This file is exclusively intended for Websocket messages between UI and Node or Daemon. 

Usually the place you might have been looking for in masq_lib is considered the file `utils.rs`. However, I don't think we want to go there either. Think about what this whole library serves for, it is supposed to offer code that can occur in both, the frontend side (UI) and the backend side. 

In this case it doesn't make sense to put it there because the UI doesn't use Actors at all... and it is of no use for it then. 

Please find a nice location in the `node` crate. I'm suggesting `/sub_lib/utils.rs` as probably the best one.